uk.co.compucorp.civicrm.hrcore/CRM/HRCore/CMSData/PathsDrupal.php

```diff
@@ -0,0 +1,27 @@
+<?php
+  class CRM_HRCore_CMSData_PathsDrupal implements CRM_HRCore_CMSData_PathsInterface {
+    private $contactData;
+    private $paths = [
+      'defaultImage' =>  '/%{base}/images/profile-default.png',
```

Comment:
also there is no benefits from having $paths as class property, it is not something shared between the class methods so I would rather remove it and and define each path as a separate constant instead.